chromium/browser_type.go

```diff
@@ -240,22 +239,26 @@ func (b *BrowserType) allocate(
 		return nil, err
 	}
 
-	path := opts.ExecutablePath
-	if path == "" {
-		path = b.ExecutablePath()
+	path, ok := ExecutablePath(opts.ExecutablePath, b.envLookupper, exec.LookPath)
+	if !ok {
+		return nil, ErrChromeNotInstalled
 	}
 
 	return common.NewLocalBrowserProcess(bProcCtx, path, args, dataDir, bProcCtxCancel, logger) //nolint: wrapcheck
 }
 
+// ErrChromeNotInstalled is returned when the Chrome executable is not found.
+var ErrChromeNotInstalled = errors.New("chromium is not installed on this system")
+
 // ExecutablePath returns the path where the extension expects to find the browser executable.
-func (b *BrowserType) ExecutablePath() (execPath string) {
-	if b.execPath != "" {
-		return b.execPath
+func ExecutablePath(
```

Comment:
Why is this exported?

Reply:
It was exported from `BrowserType` before. I'm fine unexporting it.

Reply:
I think unexporting would be good for now.

Reply:
See 72d3b03b.